app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,34 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = Payment.where(user_id: current_user.id)
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, 'f')
```

Comment:
I'm not sure why my previous comments don't show up anymore, but this line still looks the same to me.

Also, please remove all usage of `'f'` as a parameter. As I commented above, it's very hard to know what it means. If I show you this line:

```ruby
Ticket.total_price(@conference, current_user, 'f')
# or
Ticket.total_price(@conference, current_user, false)
```
I can kind of deduce that we are getting "the total price of tickets for this user in this conference"... but what do `'f'` or `false` mean?? I have no idea by just looking at this line. We need a clearer parameter here.

And again, please don't use `'f'` but rather `false`. You have a lot of `'f'` usage throughout this PR that you can improve :)